scripts/dts/gen_defines.py

```diff
@@ -154,7 +154,9 @@ def write_props(dev):
             continue
 
         # Skip properties that we handle elsewhere
-        if prop.name in {"reg", "interrupts", "compatible"}:
+        if prop.name in {"reg", "compatible", "status",
+                "interrupts", "interrupts-extended", "interrupt-parent",
+                "interrupt-controller", "gpio-controller"}:
```

Comment:
If you want to make it consistent with the rest of the code, the style I use is

```python
if prop.name in {"reg", "compatible", "status",
                 "interrupts", "interrupts-extended", "interrupt-parent",
                 "interrupt-controller", "gpio-controller"}:
```

(so that the second line with values isn't less indented than the first one).

This works too (four-line indent for any continuation lines):

```python
if prop.name in {
    "reg", "compatible", "status",
    "interrupts", "interrupts-extended", "interrupt-parent",
    "interrupt-controller", "gpio-controller"}:
```

Reply:
Fixed